bulk/sensor_vocab.csv

```diff
@@ -132,16 +119,12 @@ Spectral Irradiance: SPKIR Series J,Satlantic,OCR507 ICSW w/ Bioshutter II (Midr
 Spectrophotometer: OPTAA Series C,WET Labs,AC-S Deep
 Spectrophotometer: OPTAA Series D,WET Labs,AC-S
 Spectrophotometer: OPTAA Series J,WET Labs,AC-S
-Subsurface Mooring Platform Controller,SIO,Subsurface Mooring Controller
-Surface Buoy Communications and Power Manager,WHOI,CPM
-Surface Buoy Data Concentrator Logger,WHOI,DCL
 Surface Wave Spectra: WAVSS Series A,AXYS Technologies,TRIAXYS Directional Wave Sensor
 "Tidal Seafloor Pressure (Tsunameter 2,000 psia): PREST Series B",Sea-Bird Electronics,SBE 54
 "Tidal Seafloor Pressure (Tsunameter 6,000 psia): PREST Series A",Sea-Bird Electronics,SBE 54
 Underwater Digital Still Camera: CAMDS Series A,Kongsberg Underwater Technology Inc,Custom
 Underwater Digital Still Camera: CAMDS Series B,Kongsberg Underwater Technology Inc,Custom
 Underwater Digital Still Camera: CAMDS Series C,Kongsberg Underwater Technology Inc,Custom
-Mobile Asset Controller for Coastal AUV,Hydroid,AUV
 3-Wavelength Flourometer for AUV: FLORT Series N,WET Labs,BBFL2HYD
 Dissolved Oxygen for AUV: DOSTA Series N,Aanderaa,Optode 4330
 CTD for AUV: CTD Series N,Sea-Bird Electronics,SBE AUV Payload CTD
```

Comment:
Does this list include the NBOSI CTD?  The parser is capable of handling both types (or should be)

Reply:
This update only splits the list.  The NBOSI CTD will need to be added to the sensor bulk load spreadsheet.  The Current CTDAV Series N is the SBE.  We should use a new Series for the NBOSI CTD.